examples/README.md

```diff
@@ -232,6 +233,39 @@ partition runs on compute optimized nodes of type `cs-standard-60`. The
 [Other operating systems]: https://github.com/SchedMD/slurm-gcp/blob/master/docs/images.md#supported-operating-systems
 [slurm-gcp-v5-ubuntu2004.yaml]: ../community/examples/slurm-gcp-v5-ubuntu2004.yaml
 
+#### Usage of Node Groups
+This example defines partitions with more than one node group each. For more
+information on node groups and why they are used, see the documentation in the
+[schedmd-slurm-gcp-v5-node-group] module documentation. Some reference commands
+are listed here for specifying not only the partition, but also the correct node
+group when executing a Slurm command on a cluster generated by this blueprint.
+
+Partition: compute; Node Group: c2s30; Machine Type: c2-standard-30
+
+```bash
+srun -N 4 -p compute -w highioslur-compute-c2s30-[0-3] hostname
```

Comment:
I would recommend changing the -N 4 to -N 1, to indicate you create optionality (it can pick from any of these nodes). In fact, could you select any node of the partition?  for instance: 
``` 
srun -N 2 -p compute -w highioslur-compute-c2s30-* hostname
```